### DISCUSSION TITLE
Episode: Steve Sewell - builder.io, Partytown, Qwik

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#64] [Announcements]
## Background:

[Twitter](https://twitter.com/Steve8708)
[Builder.io](https://www.builder.io)
[Partytown](https://partytown.builder.io)
[Qwik](https://github.com/builderio/qwik)

## Intro

Andrew: Hello, welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin. 

Justin: 

## Questions

- [x] What is builder.io and what sets it apart from other WYSIWYG website builders?
- [x] Why did you build Qwik and how is it different than other web frameworks?
  - [x] How exactly does it make page load time so quick?
  - [x] What is qwik's strategy for quick loading? What things does Qwik lazy load that other frameworks might not?
  - [x] I've heard that Qwik approaches partial hydration in a unique way? 
  - [x] What is "Resumable vs. Replayable" and how does it effect the user experience? Does it change how you write the code?
  - [x] What is an entity? How does it differ from a component?
- [x] Partytown is a drastic departure from how JS is normally executed in a website. Why did you build it? What are the benefits?
  - [x] How do you handle DOM access given that web worker message passing is async?
  - [x] What are some downside to how partytown executes JS?
  - [x] Would you want to use this for _first party_ JS?
  - [x] It seems magical, when would I not want to use this?
- [x] Your approach to rendering web apps is pretty unique, given that what do you think that the future of the web will look like?  

## Tooltips

Add them in the comments below!

## Outro

Andrew: Well that’s it for this week’s episode of devtools.fm. Be sure to follow us on YouTube and wherever you consume your podcasts. Thanks for listening!

**Comment by hipstersmoothie** [comment]
- https://peeky.dev
- https://corset.dev

**Comment by just-be-dev** [comment]
- https://www.lovehulten.com/
- https://blockprotocol.org/
- https://github.com/lirantal/lockfile-lint

**Comment by steve8708** [comment]
- https://www.builder.io/c/performance-insights
